src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/bitbucket/BitbucketPullRequestDecorator.java

```diff
@@ -149,9 +158,9 @@ private void updateAnnotations(String project, String repo, AnalysisDetails anal
                             toBitbucketType(componentIssue.getIssue().type()));
                 }).collect(Collectors.groupingBy(s -> chunkCounter.getAndIncrement() / DEFAULT_MAX_ANNOTATIONS, toSet()));
 
-        for (Set<Annotation> annotations : annotationChunks.values()) {
+        for (Set<CodeInsightsAnnotation> annotations : annotationChunks.values()) {
             try {
-                client.createAnnotations(project, repo, analysisDetails.getCommitSha(), new CreateAnnotationsRequest(annotations));
+                client.get().uploadAnnotations(project, repo, analysisDetails.getCommitSha(), annotations);
```

Comment:
Client simplification:
```suggestion
                client.uploadAnnotations(project, repo, analysisDetails.getCommitSha(), annotations);
```